bench/mjolnir/buildtiles.cc

```diff
@@ -0,0 +1,55 @@
+#include <benchmark/benchmark.h>
+
+#include "baldr/rapidjson_utils.h"
+#include "filesystem.h"
+#include "mjolnir/util.h"
+
+using namespace valhalla::mjolnir;
+
+namespace {
+
+void BM_SomeFunction(benchmark::State& state, const std::string& pbf_path) {
+  valhalla::midgard::logging::Configure({{"type", ""}});
+
+  std::string const kConfigFilePath = VALHALLA_SOURCE_DIR "bench/mjolnir/config.json";
+  boost::property_tree::ptree config;
+  rapidjson::read_json(kConfigFilePath, config);
+  config.get_child("mjolnir").erase("tile_extract");
+  config.get_child("mjolnir").erase("tile_url");
+
+  BuildStage const start_stage_str = string_to_buildstage("initialize");
```

Comment:
Style nit: some place we have the `const` qualifier before the type and at other places its after (eg in function args & for loop). Maybe have the form "`const` <Type> var" everywhere? (I think thats how Valhalla codebase uses it in most places).